index.js

```diff
@@ -9,7 +9,7 @@ const decamelize = string => {
 		.replace(/([a-z\d]+)([A-Z]{2,})/g, '$1 $2')
 
 		.replace(/([a-z\d])([A-Z])/g, '$1 $2')
-		.replace(/([A-Z]+)([A-Z][a-z\d]+)/g, '$1 $2');
+		.replace(/([A-Z]+)([A-Z][a-rt-z\d]+)/g, '$1 $2');
```

Comment:
This needs a code comment. `a-rt-z` is not clear when reading it.

Reply:
I tried to keep in minimal, inspired by a previous comment I saw in the file itself. Apologies if it's not to your liking. If you have a suggested phrasing I will happily replace! 👍